src/daal4py.cpp

```diff
@@ -330,16 +331,16 @@ daal::data_management::NumericTablePtr make_nt(PyObject * obj)
 		    /*
 		     * Input is 2D F-contiguous array: represent it as SOA numeric table
 		     */
-		    daal::data_management::SOANumericTable * soatbl = NULL;
+		    daal::data_management::SOANumericTablePtr soatbl;
 
 		    // iterate over columns
 		    PyArrayIterObject * it = (PyArrayIterObject *) PyArray_IterAllButAxis(obj, &_axes);
 		    if (it == NULL) {
-			Py_XDECREF(it);
-			throw std::runtime_error("Creating DAAL SOA table from F-contigous NumPy array failed: iterator could not be created");
+                Py_XDECREF(it);
+                throw std::runtime_error("Creating DAAL SOA table from F-contigous NumPy array failed: iterator could not be created");
```

Comment:
Wrong alignment

Reply:
Looks like we should do refactoring of the alignment in whole file. But if we will do it right now then it will be very hard to look into changes related to goal of this PR. It should be done later in separate PR.